File: Tier2/src/main/java/via/sep3/group11/tier2/shared/DTOs/HousingIdDTO.java
package via.sep3.group11.tier2.shared.DTOs;

public class HousingIdDTO {
    private Long housingId;
    private String hostEmail;
    private String errorMessage;

    public HousingIdDTO(Long housingId, String hostEmail, String errorMessage) {
        this.housingId = housingId;
        this.hostEmail = hostEmail;
        this.errorMessage = errorMessage;
    }
}

File: Tier1/Shared/DTOs/HousingIdDTO.cs
namespace Shared.DTOs;

public class HousingIdDTO
{
    public long HousingId;
    public string HostEmail;
    public string ErrorMessage;

    public HousingIdDTO(long housingId, string hostEmail, string errorMessage)
    {
        HousingId = housingId;
        HostEmail = hostEmail;
        ErrorMessage = errorMessage;
    }
}